### ISSUE TITLE
train error

### ISSUE THREAD

**Issue by YIHTION** [cizhenshi/TianchiGuangdong2019_2th#2]
I am new for this. When I run train_model1.sh , it has an error. can you give me some tips? thanks!

THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=383 error=11 : invalid argument
/pytorch/aten/src/THCUNN/ClassNLLCriterion.cu:56: void ClassNLLCriterion_updateOutput_no_reduce_kernel(int, THCDeviceTensor<Dtype, 2, int, DefaultPtrTraits>, THCDeviceTensor<long, 1, int, DefaultPtrTraits>, THCDeviceTensor<Dtype, 1, int, DefaultPtrTraits>, Dtype *, int, int) [with Dtype = float]: block: [0,0,0], thread: [32,0,0] Assertion `cur_target >= 0 && cur_target < n_classes` failed.
/pytorch/aten/src/THCUNN/ClassNLLCriterion.cu:56: void ClassNLLCriterion_updateOutput_no_reduce_kernel(int, THCDeviceTensor<Dtype, 2, int, DefaultPtrTraits>, THCDeviceTensor<long, 1, int, DefaultPtrTraits>, THCDeviceTensor<Dtype, 1, int, DefaultPtrTraits>, Dtype *, int, int) [with Dtype = float]: block: [0,0,0], thread: [33,0,0] Assertion `cur_target >= 0 && cur_target < n_classes` failed.

**Comment by cizhenshi** [comment]
I'm sorry, I haven't come across this kind of error.  It seems like there's a category number mismatch. Have you modified our configuration file?

**Comment by YIHTION** [comment]
I did not modify the config files，only changed the learning rate follow your instructions。And I made the .pkl files which the training needed by myself，also I modified the PrepareData.py file to get train_0.json and val_0.json. Maybe something is wrong in these operations？
Thank you for your reply！
Best wishes！
